src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -498,5 +500,65 @@ private static void ValidateRoleDefinition(PSRoleDefinition roleDefinition)
                 throw new ArgumentException(ProjectResources.InvalidActions);
             }
         }
+        public static void ValidateScope(string scope)
+        {
+
+            if (scope.Length == 0 || !scope.StartsWith("/"))
```

Comment:
no. 

---
In reply to: [119192092](https://github.com/Azure/azure-powershell/pull/4019#discussion_r119192092) [](ancestors = 119192092)